README.md

```diff
@@ -4,32 +4,39 @@
 
 - [Getting Started](#getting-started)
 - [Usage](#usage)
-  - [Run website](#run-the-website)
-  - [Build website](#build-the-website)
-  - [Run built website](#run-built-website)
+  - [Run the website](#run-the-website)
+  - [Run Storybook](#run-storybook)
+  - [Build the project](#build-the-project)
+  - [Build the website](#build-the-website)
+  - [Build Storybook](#build-storybook)
+  - [Run the built website](#run-the-built-website)
+  - [Run built Storybook](#run-built-storybook)
   - [Clean Gatsby cache](#clean-gatsby-cache)
 - [Project Structure](#project-structure)
-- [Code Style](#code-style)
-  - [ESLint](#eslint)
-  - [Prettier](#prettier)
-  - [VS Code](#vs-code)
+- [Component Folder Structure](#component-folder-structure)
+  - [Each component includes](#each-component-includes)
+  - [Each component optionally may include](#each-component-optionally-may-include)
+  - [Example structure](#example-structure)
+- [Commits](#commits)
+- [VS Code](#vs-code)
+- [Storybook](#storybook)
 - [Style Variables](#style-variables)
 
 ## Getting Started
 
-1. Clone this repository
+1. **Clone this repository**
 
-```bash
-git clone git@github.com:pixel-point/gatsby-starter.git
-```
+    ```bash
+    git clone git@github.com:pixel-point/gatsby-starter.git
+    ```
 
-2. Install dependencies
+1. **Install dependencies**
 
-```bash
-npm install
-```
+    ```bash
+    npm install
+    ```
 
-3. Copy .env.example and rename it into .env
+1. **Copy .env.example and rename it into .env**
```

Comment:
I suppose we can provide bash snippet for that as well:

```bash
cp .env.example .env
```

Reply:
@BolotskyDev crazy man :)

Reply:
Maybe we could create an npm package that will set up gatsby project based on this starter, then automatically go to the project directory and run this bash script so the user wouldn't need to care about it? 🤔

Reply:
Nah, that npm package is clearly an overkill, we can use something a bit simpler, like the following bash script:
```bash
#!/bin/bash

# Simple bash script that allows you to initiate Pixel Point's Gatsby starter without hustle

# repo ssh path
SSH_PATH=git@github.com:pixel-point/gatsby-starter.git

# repo https path
HTTPS_PATH=https://github.com/pixel-point/gatsby-starter.git

# project default folder name
FOLDER_NAME_DEFAULT=pp-new-project

# helper variables
CHOSEN_PATH=
CHOSEN_FOLDER_NAME=

# helper initiation function

helper_initiate() {
 git clone "$1" "$2" && cd "$2" && npm i && cp .env.example .env && code . && echo "Done!"
}

# main initiation function

main_initiate() {

echo "Hello, fellow! I will help you to initiate Pixel Point's Gatsby Starter in no time! A couple of questions before we begin:"

echo "1. Do you want to use HTTPS (requires git credentials) or SSH (requires registered SSH token) protocol?"

options=("HTTPS" "SSH")

while : #infinite loop. be sure to break out of it when a valid choice is made
do
    i=1
    #we recreate manually the menu here
    for o in  "${options[@]}"; do
        echo "$i) $o"
        let i++
    done

    read reply
    #the user can either type the option number or copy the option text
    case $reply in
        "1") CHOSEN_PATH="$HTTPS_PATH"; break;;
        "2") CHOSEN_PATH="$SSH_PATH"; break;;
        *) echo "Invalid choice. Please choose an existing option number.";;
    esac

done

KEY=$reply-1

echo "Ok, let's go with \"${options[$KEY]}\""

echo "2. What is the folder name you want me to put the starter to?"
read -e -p "Enter the folder name: " FOLDER_NAME
echo "Initiating the project..."

if [[ $FOLDER_NAME ]]; then
    CHOSEN_FOLDER_NAME="$FOLDER_NAME"
else
    CHOSEN_FOLDER_NAME="$FOLDER_NAME_DEFAULT"
fi
}

main_initiate

helper_initiate "$CHOSEN_PATH" "$CHOSEN_FOLDER_NAME"
```
Feel free to try it!

Reply:
@BolotskyDev you are crazy for sure :D